samples/cassandra/src/main/java/sample/config/CassandraConfig.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2014-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sample.config;
+
+import javax.annotation.PreDestroy;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.Session;
+import com.datastax.driver.core.exceptions.AlreadyExistsException;
+
+import org.cassandraunit.utils.EmbeddedCassandraServerHelper;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import org.springframework.data.cassandra.core.CassandraOperations;
+import org.springframework.data.cassandra.core.CassandraTemplate;
+import org.springframework.session.data.cassandra.CassandraSessionRepository;
+
+/**
+ * Example Cassandra initialization code. In a very similar way as the integration test suite,
+ * {@code cassandra-unit} is leveraged to bring up a test cluster to connect to. Here you would
+ * normally inject your Cassandra cluster connection information, as well as perform any kind of
+ * schema pre-checks and initialization.
+ *
+ * @author Jesus Zazueta.
+ */
+@Configuration
+public class CassandraConfig {
+
+	static final String CREATE_KEYSPACE = "CREATE KEYSPACE spring_session WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};";
```

Comment:
How about changing to `CREATE KEYSPACE IF NOT EXISTS ...`? Removes the need to catch `AlreadyExistsException` in `public Session session()`.